### PR TITLE
feat(bank linking): added additional info line in case that user is about to access add a bank

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagas.ts [hunk 1]
```diff
@@ -66,9 +66,6 @@ export default ({ api }: { api: APIType }) => {
       const locks: ReturnType<typeof api.getWithdrawalLocks> = yield call(
         api.getWithdrawalLocks
       )
-      // eslint-disable-next-line
-      console.log('locks', locks)
-
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagas.ts:71]: left over removed

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagas.ts:71]: @milan-bc did you see my note about not needing to use // eslint-disable-next-line when running the app locally?

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/TransferDetails/template.success.tsx [hunk 2]
```diff
@@ -179,6 +179,22 @@ const Success: React.FC<Props> = props => {
                 />
               )}
             </TopText>
+
+            {props.addBank && (
+              <Text size='16px' weight={500} color='grey600'>
+                <FormattedMessage
+                  id='modals.simplebuy.transferdetails.bank_link_info'
+                  defaultMessage='To link your bank, send {simbol}1 or more to your {currency} wallet.'
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/TransferDetails/template.success.tsx:187]: symbol

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/TransferDetails/template.success.tsx [hunk 3]
```diff
@@ -179,6 +179,22 @@ const Success: React.FC<Props> = props => {
...
+                  values={{
+                    currency: props.account.currency,
+                    simbol:
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/TransferDetails/template.success.tsx:190]: symbol

File: packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagas.ts (hunk 1 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagas.ts:71]: yes that's why I removed this one seems like this was left by accident :/

**Comment by Ogungodspower5** [comment]
Bank account::prospect bank
Client name::Barnett R Hilda 
Account number; 5705468
Routing number :071102827
Bank address :141 veterans parkway Watseka Illinois 60970
Home address :: 104 W Herron Ct
Watseka

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/TransferDetails/template.success.tsx (hunk 3 above)
Comment (Ogungodspower5) [reply packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/TransferDetails/template.success.tsx]: Bank account::prospect bank
Client name::Barnett R Hilda 
Account number; 5705468
Routing number :071102827
Bank address :141 veterans parkway Watseka Illinois 60970
Home address :: 104 W Herron Ct
Watseka

File: packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagas.ts (hunk 1 above)
Comment (Ogungodspower5) [reply packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagas.ts:71]: Bank account::prospect bank
Client name::Barnett R Hilda 
Account number; 5705468
Routing number :071102827
Bank address :141 veterans parkway Watseka Illinois 60970
Home address :: 104 W Herron Ct
Watseka

Comment (Ogungodspower5) [reply packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagas.ts:71]: Credit me in my account

**Review by Ogungodspower5 — COMMENTED** [review]
Bank account::prospect bank
Client name::Barnett R Hilda 
Account number; 5705468
Routing number :071102827
Bank address :141 veterans parkway Watseka Illinois 60970
Home address :: 104 W Herron Ct
Watseka credit my account

**Comment by milan-bc** [comment]
@schnogz this should be done, please do re-review

**Review by schnogz — APPROVED** [review]

**Pull request merged**